service/src/main/java/greencity/service/notification/NotificationServiceImpl.java

```diff
@@ -73,10 +69,18 @@ public class NotificationServiceImpl implements NotificationService {
      */
     @Override
     public void notifyUnpaidOrders() {
-        for (Order order : orderRepository.findAllByOrderPaymentStatus(OrderPaymentStatus.UNPAID)) {
+        List<Order> orderList = orderRepository.findAllByOrderPaymentStatus(OrderPaymentStatus.UNPAID);
+
+        orderList = orderList.stream().filter(order -> {
+            Duration differenceBetween = Duration.between(order.getOrderDate(), LocalDateTime.now(clock));
+            return differenceBetween.toDays() < 24;
```

Comment:
please move it to the method

Reply:
We will send notifications if the order was submitted less than 24 days before and is not paid yet
What do we do with orders if they are not paid for 24 days?

Reply:
I asked to Vicka and she told me that after 24 days nothing won´t happen. I'll ask her again

Reply:
> please move it to the method

In this case, I should to create a Order Service to filter them? , because orderRepository.findAllByOrderPaymentStatus is from the repository

Reply:
> We will send notifications if the order was submitted less than 24 days before and is not paid yet What do we do with orders if they are not paid for 24 days?

I asked to Vicka and she told me that after 24 days nothing won´t happen. I'll ask her again